internal/contracts/contracts.go

```diff
@@ -284,9 +284,10 @@ func (mc *JSONContract) Unmarshal() (Contract, error) {
 func GenerateRandomContract() (*Contract){
 	min := 1
 	maxVer := 65535
-	b := make([]byte, 32)
-	mrand.Read(b)
-	genRecipPubKeyHash := b 
+	b := make([]byte, 8)
+	binary.LittleEndian.PutUint64(b, mrand.Uint64())
+
+	genRecipPubKeyHash := hashing.New(b)
```

Comment:
`b` isn't random

Reply:
I tested it out on playground, and it generates a random []byte every time I call the function